code/modules/mob/living/grabbing.dm

```diff
@@ -809,9 +809,11 @@
 		return
 	if(ishuman(C))
 		var/mob/living/carbon/human/H = C
-		if(istype(H.wear_neck, /obj/item/clothing/neck/psycross/silver))
-			to_chat(user, span_userdanger("SILVER! HISSS!!!"))
-			return
+		for(var/I in C.contents)
+			if(SSenchantment.has_enchantment(I, /datum/enchantment/silver))
```

Comment:
this also means holding anything silver protects you from grabs which is a little much i think

Reply:
could check the grabbed zone's item instead so if they have a silver psycross around their neck you can't strangle them

Reply:
it is not a grab protect, it protects the user from having you blood sucked:

<img width="1330" height="517" alt="Image" src="https://github.com/user-attachments/assets/fc391cfc-babf-4cb1-8834-5f1bc4ba24aa" />

Reply:
"They are wearing my BANE!" only appears after i try to drink from their throat or another place, i can grab someone with a psycross with no issues